.github/workflows/ci-coverage.yml

```diff
@@ -0,0 +1,21 @@
+name: initial-coverage
```

Comment:
We need to upload some coverage as a baseline so there's something to compare carry forward reports with.

We can remove this after a report has been generated on `main`